kitsune/messages/urls.py

```diff
@@ -1,9 +1,16 @@
-from django.urls import re_path
+from django.urls import include, re_path
 
-from kitsune.messages import views
+from kitsune.messages import api, views
+
+api_patterns = [
+    re_path(
+        r"^usernames_or_groups", api.usernames_or_groups, name="messages.api.usernames_or_groups"
```

Comment:
I would rephrase this to `autocomplete` or something like that. Similar comment about the method. I also added a comment in the api.py to use directly the `get_autocomplete_suggestions`. Personally I find it more descriptive and suitable to focus on the autocomplete functionality in the url and not on the type that's returned

Reply:
Agreed.